src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -490,6 +491,14 @@ export default {
 			this.handleScroll()
 			this.handleUnreadMention()
 		}, 50),
+
+		// Route to the first search result
+		handleEnter() {
+			if (this?.conversationsList[0]?.token) {
+				this.$router.push({ name: 'conversation', params: { token: this.conversationsList[0].token } })
```

Comment:
return after this?

Reply:
Why? There's no need to return anything there

Reply:
Okay, seems I misunderstood the part after the if